server/@types/forms/index.d.ts

```diff
@@ -165,6 +165,8 @@ declare module 'reviewPlanForms' {
   export interface WhoCompletedReviewForm {
     completedBy: ReviewPlanCompletedByValue
     completedByOther?: string
```

Comment:
Really minor! (sorry!),
but did you also mean to remove the original `completedByOther` field from this form object?

If we can get rid of it (as we have done in the DTO) that would be great, as I cant see anywhere else that is using it